Code/GraphMol/Wrap/EditableMol.cpp

```diff
@@ -97,17 +104,27 @@ struct EditableMol_wrapper {
              "Remove the specified bond from the molecule")
 
         .def("AddBond", &EditableMol::AddBond,
-             (python::arg("mol"), python::arg("beginAtomIdx"),
+             (python::arg("beginAtomIdx"),
```

Comment:
They aren't necessary and we don't use them in other places.  I removed them mainly because I was tracing a bug that turned out to be unrelated and tried removing them to fix it.

We tend to have inconsistent usage of this:

```
        .def("GetNumBonds", &ROMol::getNumBonds,
             (python::arg("onlyHeavy") = true),
```

and

```
.def("AddConformer", AddMolConformer,
             (python::arg("self"), python::arg("conf"),
              python::arg("assignId") = false),
```

I don't particularly care about the style, however I would either (a) always use the "self" name for some explicitness or (b) just drop it completely.

I can revert back if desired.